src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -41,19 +40,19 @@
 
   static final long serialVersionUID = 1L;
 
-  private Datastore datastore;
+  private DatastoreOptions datastoreInstance;
   private UserDao dao;
 
   private static final String CURRENT_USER_ARG = "current";
   private static final String GARDEN_LIST_ARG = "garden-list";
   private static final String GARDEN_ADMIN_LIST_ARG = "garden-admin-list";
-  private static final String CURRENT_USER_KEY = "0";
+  private static final String CURRENT_USER_KEY = "1";
 
   /** Initializes the servlet. Connects it to Datastore. */
   @Override
   public void init() throws ServletException {
-    this.datastore = DatastoreOptions.getDefaultInstance().getService();
-    this.dao = new UserDao(datastore);
+    this.datastoreInstance = DatastoreOptions.getDefaultInstance();
```

Comment:
Why are you storing the datastore as an ivar?

Reply:
As of now, this is unneeded. Thus I will remove it.